pkg/storages/swift/folder.go

```diff
@@ -32,7 +32,7 @@ func NewFolder(connection *swift.Connection, container swift.Container, path str
 	return &Folder{connection, container, path}
 }
 
-func ConfigureFolder(prefix string, settings map[string]string) (storage.Folder, error) {
+func ConfigureFolder(prefix string, settings map[string]string, method ...string) (storage.Folder, error) {
```

Comment:
Why folder should know anything about method? We need to pass the custom behaviour in the wal_push command handler instead.